src/elements/helpers/getImageSrc.ts

```diff
@@ -13,7 +13,10 @@ export const getImageSrc = (assets: Asset[], desiredWidth: number) => {
     widthDifference(stringOrNumberToNumber(assetB.fields.width));
 
   const sortedAssets = assets
-    .filter((asset) => !asset.fields.isMaster)
+    .filter(
+      (asset) =>
+        asset.fields.isMaster === false || asset.fields.isMaster === undefined
```

Comment:
It looks like `isMaster` can also be a [string](https://github.com/guardian/prosemirror-elements/blob/ef306e9a94a3983452a9bdac5d671843f08224fd/src/elements/helpers/defaultTransform.ts#L13). Should we include another condition?

```suggestion
        asset.fields.isMaster === "false" || asset.fields.isMaster === false || asset.fields.isMaster === undefined
```

Reply:
Not sure if that ever actually happens, but safer to check for it!